sympy/solvers/bivariate.py

```diff
@@ -181,7 +181,10 @@ def _lambert(eq, x):
     for arg in args:
         for k in lambert_real_branches:
             w = LambertW(arg, k)
-            if k and not w.is_real:
+            print(w.is_real, len((w).atoms(Symbol)), len((w).atoms(x)), w)
```

Comment:
Looks like you left some debug code here

Reply:
Oh, sorry for this!